src/libraries/System.Diagnostics.TraceSource/ref/System.Diagnostics.TraceSource.cs

```diff
@@ -194,7 +194,7 @@ public enum TraceEventType
     public abstract partial class TraceFilter
     {
         protected TraceFilter() { }
-        public abstract bool ShouldTrace(System.Diagnostics.TraceEventCache cache, string source, System.Diagnostics.TraceEventType eventType, int id, string formatOrMessage, object[] args, object data1, object[] data);
+        public abstract bool ShouldTrace(System.Diagnostics.TraceEventCache? cache, string source, System.Diagnostics.TraceEventType eventType, int id, string? formatOrMessage, object[]? args, object? data1, object[]? data);
```

Comment:
Same question about these `object[]?`s... should they be `object?[]?`?

Reply:
yep